.github/workflows/ci.yml

```
@@ -0,0 +1,49 @@
+name: Java SDK CI
+
+on:
+  pull_request:
+    branches:
+      - 'main'
+      - 'develop'
+  push:
+    branches:
+      - 'main'
+      - 'develop'
+
+jobs:
+  build_and_test_library:
+    name: Build & Test Library
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Set up JDK 8 with Gradle Cache
+        uses: actions/setup-java@v4
+        with:
+          java-version: '8'
+          distribution: 'corretto'
+          cache: 'gradle'
+
+      - name: Build and Test the Library
+        run: ./gradlew build
```

At least according to README `install` should be used for the library.
```suggestion
        run: ./gradlew install
```

---

changed and uploaded to artifacts
previously there was a hard dependency between examples and library introduced as part of my change which is now deleted and we simply use artifacts for running the examples build/install